### PR TITLE
Listing Component Mobile

### PR DESCRIPTION
**Pull request by blazeque**
| - | - |
| --- | --- |
| Decisions |  Fixing Listing component to mobile and Tablet |
| Animated GIF | tab-responsive |

### CODE REVIEW

File: vue/components/ui/listing-platforme/listing-platforme.stories.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ storiesOf("Data", module)
         },
         template: `
             <div>
+            <global-platforme />
```
Comment (gcandal) [inline vue/components/ui/listing-platforme/listing-platforme.stories.js]: Fix the indentation

File: vue/components/ui/listing-platforme/listing-platforme.stories.js [hunk 2]
```diff
@@ -63,8 +64,8 @@ storiesOf("Data", module)
                     v-bind:filter-fields="filterFields"
                 >
                     <template v-slot:icons>
-                        <img v-bind:src="img" v-bind:style="imgStyle" />
-                    </template>
+                        <img class="logo" v-bind:src="img" v-bind:style="imgStyle" />
+                    </template>
```
Comment (gcandal) [inline vue/components/ui/listing-platforme/listing-platforme.stories.js]: Remove the extra whitespace

File: vue/components/ui/listing-platforme/listing-platforme.vue [hunk 3]
```diff
@@ -57,13 +57,13 @@
 }
 
 .scroll-button {
-    background: #ffffff url("~./assets/arrow-up.svg") no-repeat center;
+    background: $white url("~./assets/arrow-up.svg") no-repeat center;
```
Comment (gcandal) [inline vue/components/ui/listing-platforme/listing-platforme.vue]: Can you please remove these color fixes? We'll fix it in every component in a different PR, as soon as the colors PR is merged.

File: vue/components/ui/listing-platforme/listing-platforme.vue [hunk 4]
```diff
@@ -259,12 +259,59 @@ input[type="text"] {
...
 input[type="text"]:focus {
-    background-color: #ffffff;
-    border-color: #aaaaaa;
+    background-color: $white;
+    border-color: $light-grey;
+}
+
+body.mobile .listing > .container-platforme,
+body.tablet .listing > .container-platforme {
+    padding: 10px;
```
Comment (gcandal) [inline vue/components/ui/listing-platforme/listing-platforme.vue]: Use 4-item version

File: vue/components/ui/listing-platforme/listing-platforme.vue [hunk 5]
```diff
@@ -259,12 +259,59 @@ input[type="text"] {
...
+}
+
+body.mobile .listing > .container-platforme > .container-header,
+body.tablet .listing > .container-platforme > .container-header {
+    padding: 0;
```
Comment (gcandal) [inline vue/components/ui/listing-platforme/listing-platforme.vue]: Use 4-item version + `px`

File: vue/components/ui/listing-platforme/listing-platforme.vue [hunk 6]
```diff
@@ -259,12 +259,59 @@ input[type="text"] {
...
+    width: 100%;
+}
+
+body.mobile .listing > .container-platforme > .container-header > .container-header-right ::v-deep,
```
Comment (gcandal) [inline vue/components/ui/listing-platforme/listing-platforme.vue]: `::v-deep` is doing nothing here

File: vue/components/ui/listing-platforme/listing-platforme.vue [hunk 7]
```diff
@@ -259,12 +259,59 @@ input[type="text"] {
...
+body.tablet .listing > .container-platforme > .container-header > .container-header-right ::v-deep {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+body.mobile .listing > .container-platforme > .container-header > .container-header-right ::v-deep > img.logo,
```
Comment (gcandal) [inline vue/components/ui/listing-platforme/listing-platforme.vue]: I don't think you need the `::v-deep`

Comment (blazeque) [reply vue/components/ui/listing-platforme/listing-platforme.vue]: I made tests. and worked only with the v-deep

**Pull request closed without merging**